### DISCUSSION TITLE
欢迎来到 fcitx 5 讨论区

### DISCUSSION THREAD

**Discussion by wengxt** [fcitx/fcitx5#278] [General]
可以在此提问，讨论，或者分享自己的点子。

bug 汇报区（Issues）已经被太多人用来问问题，但是 Google Group（仍会继续使用）对大多数人来说并不好用。

可以在此询问关于配置，安装，自定义的问题。

(因为大部分开发者都是中国人，所以可以用中文提问。)

**Comment by englianhu** [comment]
[Ask a Favor : KDE Plasma 5 Input Methods](https://d.cosx.org/d/422243-ask-a-favor-kde-plasma-5-input-methods)

**Comment by xingwozhonghua126** [comment]
建议fcitx5可以让各个输入法私有化kimpanel接口，私有化的输入法kimpanel可以在当前输入法内部生效。

建议kimpanel中只包含输入法候选词部分。输入法任务栏，应该用经典界面的。输入法状态栏（fcitx术语中的主窗口）由各个输入法提供。（框架需要规定默认位置）

就是把fcitx-class-ui分成三个部分，候选词，输入法状态栏（fcitx5已经去掉），输入法任务栏。

因为候选词窗口是经常被各个输入法替换的，输入法任务栏不希望被替换掉，如果替换掉切换输入法会显示的样式不一样，很奇怪。
